server/tests/common/rpc.py

```diff
@@ -496,6 +496,15 @@ class BaseAuthenticatedAccountRpcClient:
     async def _do_request(self, req: bytes, family: str) -> bytes:
         raise NotImplementedError
 
+    async def account_delete_send_validation_token(
+        self,
+    ) -> authenticated_account_cmds.latest.account_delete_send_validation_token.Rep:
+        req = authenticated_account_cmds.latest.account_delete_send_validation_token.Req()
+        raw_rep = await self._do_request(req.dump(), "authenticated_account")
```

Comment:
Unrelated to this PR: we should use constants (or enum) for command families here 
```python
AUTHENTICATED_ACCOUNT="authenticated_account"
...
raw_rep = await self._do_request(req.dump(), AUTHENTICATED_ACCOUNT)
```